src/main/java/fr/xephi/authme/listener/PlayerListener.java

```diff
@@ -103,103 +98,104 @@
     @Inject
     private BungeeSender bungeeSender;
```

Comment:
This field is now unused. The logger as well.
Also update the test please not to have bungee sender as mock field